### PR TITLE
chore/Total approved cost is zero if you get a validation error

### PR DESCRIPTION
**Pull request by hvekriya**
Total approved cost is zero if you get a validation error #206

## Checklist

- [ ] Unit tests
- [ ] Code coverage checked
- [x] Coding standards
- [x] Error Handling
- [ ] Security/performance considered?
- [ ] Deployment changes considered?

### CODE REVIEW

File: app/assets/javascripts/view-claim.js [hunk 1]
```diff
@@ -24,6 +24,12 @@ $(function () {
   $('input.approved-amount').on('input', function () {
     totalApproved()
   })
+  $('.claim-expense-status').each(function () {
+    if (this[this.selectedIndex].value === 'APPROVED') {
```
Comment (stevenalexander) [inline app/assets/javascripts/view-claim.js]: there is also status APPROVED_DIFF_AMOUNT and MANUALLY_PROCESSED

**Comment by hvekriya** [comment]
Changes made. Updated the code to also work with approved-diff-amount and manually-processed

**Comment by hvekriya** [comment]
Improved the function to deal with 'Should the total approved cost also consider any deductions? #220'

**Comment by pwright08** [comment]
should also fix https://github.com/ministryofjustice/apvs-asynchronous-worker/issues/123

**Pull request merged**